### ISSUE TITLE
Error solving the challenge. string index out of range

### ISSUE THREAD

**Issue by KazadorFR** [FlareSolverr/FlareSolverr#789]
### Have you checked our README?

- [X] I have checked the README

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version:3.2.0
- Last working FlareSolverr version: none
- Operating system:ubuntu server 22.04
- Are you using Docker: yes
- FlareSolverr User-Agent (see log traces or / endpoint):"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36
- Are you using a VPN: Only for qbittorent
- Are you using a Proxy: no
- Are you using Captcha Solver: no
- If using captcha solver, which one:
- URL to test this issue:https://cpasbiens911.fr/
```

### Description

Unable to solve the challenge for this indexer. I have tried multiple methods, both using Docker and a regular installation. I have tried different IP addresses as well. After 2 days of work, I am unable to solve the problem.

### Logged Error Messages

```text
2023-05-30 13:57:30 INFO     Incoming request => POST /v1 body: {'postData': '', 'maxTimeout': 55000, 'cmd': 'request.post', 'url': 'https://cpasbiens911.fr/'}
2023-05-30 13:57:30 ERROR    Error: Error solving the challenge. string index out of range
2023-05-30 13:57:30 INFO     Response in 0.525 s
2023-05-30 13:57:30 INFO     172.17.0.1 POST http://172.17.0.1:8191/v1 500 Internal Server Error
```

### Screenshots

![image](https://github.com/FlareSolverr/FlareSolverr/assets/42878970/604c0892-dec0-47e0-814f-512db3ad9ff3)
![image](https://github.com/FlareSolverr/FlareSolverr/assets/42878970/8efa6f72-4231-4e1b-b376-47b7a97a6f6d)
![image](https://github.com/FlareSolverr/FlareSolverr/assets/42878970/e5ac489b-4c2c-4d7d-91bf-a5d011235520)

**Comment by ilike2burnthing** [comment]
Are you able to access the site in browser from the same IP address(es)? If not, FlareSolverr won't be able to access it either.

However, the error isn't as it should be, so something needs updated there.

**Comment by KazadorFR** [comment]
> 

I tried (from Canada) to connect using my personal IP and VPN, and I was able to connect without any issues. I still can't understand the error. :/

**Comment by KazadorFR** [comment]
> Are you able to access the site in browser from the same IP address(es)? If not, FlareSolverr won't be able to access it either. I believe it's geo-restricted to French IPs.
> 
> However, the error isn't as it should be, so something needs updated there.

I can't believe it was that simple... By typing the website on Google, I saw another link ending in ".uk". I updated the address and it works now...

**Comment by ilike2burnthing** [comment]
Can you provide the link? I can find multiple fake clones, but none with a TLD of .uk

**Comment by KazadorFR** [comment]
> Pouvez-vous fournir le lien? Je peux trouver plusieurs faux clones, mais aucun avec un TLD de .uk

Indeed, it's my mistake. I apologize for the confusion earlier. The website you mentioned is "https://www.cpasbien.sk/". However, upon closer inspection of your comment, it seems that the site "https://cpasbien911.fr/" redirects you to another link, "https://cpasbien2022.fr/", which is also functional. Both sites are working correctly on your end.

**Comment by ilike2burnthing** [comment]
Like I said, they're fake clones; keyword searches don't work for either, and downloads only work for *2022.fr

Use a different French tracker if you can't use a VPN or proxy to access that one.

**Comment by Darlio** [comment]
Hi, I have same problem... But now the valid cpasbien url is, and works well as : 'https://cpasbien-vf.fr/'
I have the following error in Jackett, with a working configuration of flaresolverr : 
`Exception (cpasbien): FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Error solving the challenge. string index out of range: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Error solving the challenge. string index out of range`

Question : How do I configure either app to have a "PostData='a=b' " ?

Thanks for any help ! Best regards...

**Comment by p060477** [comment]
me too.....how to solve...??...this is a real BUG isn't it...??...
:(

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/789#issuecomment-1569231328

Also reopening as the error text was never fixed.

**Comment by p060477** [comment]
so you confirm that up to now **there is no way** to solve this BUG...??

**Comment by ilike2burnthing** [comment]
Again - https://github.com/FlareSolverr/FlareSolverr/issues/789#issuecomment-1569231328 - if that applies to you then it's not a bug.

**Comment by ilike2burnthing** [comment]
Again again, https://github.com/FlareSolverr/FlareSolverr/issues/789#issuecomment-1569231328 and not a bug.

**Comment by p060477** [comment]
> Again again, [#789 (comment)](https://github.com/FlareSolverr/FlareSolverr/issues/789#issuecomment-1569231328) and not a bug.

again and again HOW TO SOLVE...???how to finally pass the cloudflare test on that site with this flaresolverr tool..??
AGAIN if not a bug **pls give here the simple solution**
thxs

**Comment by ilike2burnthing** [comment]
The linked comment says it's geo-restricted to French IPs, so use a French IP.

**Comment by p060477** [comment]
DONE....: **not solved at all**
but have you read this:


> Hi, I have same problem... But now the valid cpasbien url is, and works well as : 'https://cpasbien-vf.fr/' I have the following error in Jackett, with a working configuration of flaresolverr : `Exception (cpasbien): FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Error solving the challenge. string index out of range: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Error solving the challenge. string index out of range`
> 
> Question : How do I configure either app to have a "PostData='a=b' " ?
> 
> Thanks for any help ! Best regards...

**Comment by ilike2burnthing** [comment]
You did that in the space of a minute? What VPN or proxy did you use? Did you try multiple servers? Were both Jackett/Prowlarr and FlareSolverr using the same VPN/proxy?

**Comment by p060477** [comment]
done it BEFORE....
have you memory leaks...??...:
https://github.com/FlareSolverr/FlareSolverr/issues/1032#issuecomment-1880180638

**Comment by ilike2burnthing** [comment]
If that was the result when using a VPN or proxy (which you didn't specify) then as you can see it's a completely different error. Provide the environment information there as originally requested, reply to my questions above there as well, and try increasing the timeout as instructed.

**Comment by p060477** [comment]
also increased....and about yr believing...here it is the screen shoot....:
![cpasbien](https://github.com/FlareSolverr/FlareSolverr/assets/12773559/09751d7c-409d-4c10-822b-9ffc68b8174b)

i still repeat the qurestion done also in the very late novembner by Darlio:
https://github.com/FlareSolverr/FlareSolverr/issues/789#issuecomment-1832241982

how to solve this **huge BUG...??...**

**Comment by ilike2burnthing** [comment]
Resolved in Jackett [v0.21.1536](https://github.com/Jackett/Jackett/releases/v0.21.1536).

**Comment by p060477** [comment]
hi now the only 3 non working indexers are:
YourBittorrent  : https://yourbittorrent.com/
TorrentFunk: https://www.torrentfunk.com/
Torlock : https://www.torlock2.com/

cause, as you have already stated, they are down
so my question is: why are in jackett if they are down...??...for what is really worth...??
only for loosing time in tryin the tests in vain...??
my question is if they are down and non working why are still suggested to be added in jackett as public indexers...??
